pages/blueprint/index.css

```diff
@@ -0,0 +1,9 @@
+.user-list {
```

Comment:
I didn't mention this yesterday, but now that `.user-list` is higher in the DOM, can you add `.pf-c-table` to this one (e.g. `.user-list .pf-c-table`)?

Reply:
I adjusted but I had to change the padding to margin, for some reason padding on the table element was not applied.